src/browser-polyfill.js

```diff
@@ -30,17 +37,26 @@ if (typeof browser === "undefined" || Object.getPrototypeOf(browser) !== Object.
      * not exist when accessed, but behaves exactly as an ordinary WeakMap
      * otherwise.
      *
-     * @param {function} createItem
-     *        A function which will be called in order to create the value for any
-     *        key which does not exist, the first time it is accessed. The
-     *        function receives, as its only argument, the key being created.
+     * @template {object} K
+     * @template V
      */
     class DefaultWeakMap extends WeakMap {
+      /**
+       * @param {function(K):V} createItem
+       *        A function which will be called in order to create the value for any
+       *        key which does not exist, the first time it is accessed. The
+       *        function receives, as its only argument, the key being created.
+       * @param {ReadonlyArray<[K, V]>} [items]
```

Comment:
Remove this `@param`. It doesn't help (perhaps we should remove the param altogether since it is never used - please do NOT make that change in this commit though).

Reply:
The `@param` should be removed. My comment might have been ambiguous, when I said "please do not make that change", I was referring to the removal of the `items` parameter from the function.

Reply:
That violates the&nbsp;<code>valid&#x2011;jsdoc</code> ESLint rule.